src/lib/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const LOG_IN_SUCCESS_KEY = 'LOG_IN_SUCCESS';
 export const LOG_OUT_SUCCESS_KEY = 'LOG_OUT_SUCCESS';
+export const PREV_PATH = 'PREV_PATH';
```

Comment:
Could you name it `CALLBACK_URL` just like next-auth do? And our storage keys should be consistent:
<img width="783" alt="image" src="https://user-images.githubusercontent.com/7880675/153342597-9724cdba-15c9-4df5-a762-536afc7cbdf2.png">
Maybe we should use `callback.url` as the key since next-auth is using this pattern, could you help refactor related keys as well?